fastlane/Gemfile.lock

```diff
@@ -1,23 +1,23 @@
 GEM
```

Comment:
The changes on this file are due to fastlane "forcing" me to update it. I also piggybacked plugins updates.